Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenImagesHelper.java

```diff
@@ -83,10 +83,10 @@ public MetadatenImagesHelper(Prefs inPrefs, DigitalDocument inDocument) {
      * delete pages from the end of pyhsicalDocStruct.
      */
     public void createPagination(Process process, URI directory) throws TypeNotAllowedForParentException, IOException {
```

Comment:
TypeNotAllowedForParentException - not used exception.

Reply:
How do you find these? Is there a way to search for them?

Reply:
I just open file in IDE and it informs me that there are declared but not thrown exceptions.